okhttp-handler/monix/src/main/scala/com/softwaremill/sttp/okhttp/monix/OkHttpMonixHandler.scala

```diff
@@ -84,10 +85,17 @@ object OkHttpMonixHandler {
       implicit s: Scheduler): SttpHandler[Task, Observable[ByteBuffer]] =
     new FollowRedirectsHandler(new OkHttpMonixHandler(client, closeClient)(s))
 
-  def apply()(implicit s: Scheduler = Scheduler.Implicits.global)
+  def apply(connectionTimeout: FiniteDuration = SttpHandler.DefaultConnectionTimeout)(
+      implicit s: Scheduler = Scheduler.Implicits.global)
     : SttpHandler[Task, Observable[ByteBuffer]] =
-    OkHttpMonixHandler(OkHttpHandler.buildClientNoRedirects(),
-                       closeClient = true)(s)
+    OkHttpMonixHandler(
+      OkHttpHandler
```

Comment:
I suppose this client-creation code could also be moved to `OkHttpHandler` object instead of duplicating it?